Themes/default/scripts/script.js

```diff
@@ -1860,7 +1855,8 @@ smc_preview_post.prototype.onDocSent = function (XMLDoc)
 		var tempHTML;
 		for (var i = 0; i < numNewPosts; i++)
 		{
-			new_replies[new_replies.length] = newPosts[i].getAttribute("id");
+			// new_replies[new_replies.length] = newPosts[i].getAttribute("id");
```

Comment:
What's with this line?

Reply:
Apparently I forgot to go back and delete it after I was done testing.

Reply:
Out of curiosity, how did you test it? I just came with that solution because it seemed legit, but I didn't know exactly how to do it.

Also, I can't seem to find intialization of new_replies. Is it needed in JS?

Reply:
> Out of curiosity, how did you test it?

In one browser window, I logged in as testuser1 and started a reply. In a different, private browser window, I logged in as testuser2 and posted a reply to the same topic. Then I went back to the first browser window and tried to post the reply. When I was sent back to the post authoring page with the usual warning message, I checked the JS console to see if the error reported in #5200 had occurred. It had not.

> I can't seem to find intialization of new_replies. Is it needed in JS?

Odd. I have a clear memory of adding `var new_replies = new Array();` to the script, but now I don't see it.

Anyway, variable declaration is a bit weird in JavaScript. If a variable is declared using the `var` keyword, it will be local to that function only. If it is declared without the `var` keyword, it will be a global variable. Meanwhile, a line like `new_replies[i] = newPosts[i].getAttribute("id");` will create the `new_replies` array implicitly if it does not already exist. So the code will work without the `var new_replies = new Array();` line, but it makes the `new_replies` variable global instead of local. It would be good form to fix that by adding the declaration.

Reply:
`var new_replies = new Array();` added in 2e48b89